python/feathub/processors/flink/table_builder/tests/table_builder_test_utils.py

```diff
@@ -95,3 +98,31 @@ def _create_file_source(
             timestamp_format=timestamp_format,
             max_out_of_orderness=timedelta(minutes=1),
         )
+
+
+class MockTableDescriptor(TableDescriptor, ABC):
+    def __init__(
+        self,
+        name: str,
+        keys: Optional[List[str]],
+        timestamp_field: Optional[str],
```

Comment:
How about set this field to None by default so that most callers don't need to specify None?